spyder/plugins/ipythonconsole/plugin.py

```diff
@@ -896,6 +896,8 @@ def register_client(self, client, give_focus=True):
             if (self.main.projects is not None and
                     self.main.projects.get_active_project() is not None):
                 cwd_path = self.main.projects.get_active_project_path()
+        elif CONF.get('workingdir', 'startup/use_fixed_directory'):
+            cwd_path = CONF.get('workingdir', 'startup/fixed_directory')

```

Comment:
Since this value may not exist this call should probably include a default

`CONF.get('workingdir', 'startup/fixed_directory', default=somethig?)`